### DISCUSSION TITLE
Change or recrate backup repository

### DISCUSSION THREAD

**Discussion by vincemd22** [nextcloud/all-in-one#5635] [Questions] [answered]
Hello,

Thank you for your work on Nextcloud All-in-One. Version v10.0.0 provides an excellent solution with an ssh repository.

Unless I'm mistaken, I didn't find this use case in the readme.

- If I want to move my BorgBackup repository elsewhere, how can I define another repository on my current instance ?

- If my BorgBackup repository has an incident, how can I define a new repository and recreate backups while keeping my current instance ?

Thanks for your time and hard work

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/discussions/5593

**Comment by vincemd22** [comment]
Thanks for your reply:

If I understand correctly, the best solution is to execute this command:

sudo docker exec nextcloud-aio-mastercontainer rm /mnt/docker-aio-config/data/borg.config
